sway/tree/view.c

```diff
@@ -1063,6 +1063,11 @@ static void view_child_handle_surface_map(struct wl_listener *listener,
 		wl_container_of(listener, child, surface_map);
 	child->mapped = true;
 	view_child_damage(child, true);
+
+    struct sway_workspace *workspace = child->view->container->pending.workspace;
+    if (workspace) {
+            wlr_surface_send_enter(child->surface, workspace->output->wlr_output);
+    }
```

Comment:
Whitespace mismatch here, should use tabs to match the rest of the file.

Reply:
Oops, thanks.